### ISSUE TITLE
Support for Magento 2.2.6 and 2.1.15

### ISSUE THREAD

**Issue by igorwulff** [tig-nl/postnl-magento2#68]
### Make sure you are using the *latest* version: https://tig.nl/buckaroo-magento-extensies/
Issues with outdated version will be rejected.
- [X ] I've **verified** and **I assure** that I'm running the latest version of the TIG Buckaroo Magento extension.

---

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with the TIG Buckaroo Magento extension)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [X] Other

---

### Description of your *issue*, suggested solution and other information

Magento released a critical security update for Magento 2.2.6 and 2.1.15. Can you create a new release and at least a RC for this as soon as possible?

**Comment by tig-rikjonkmans** [comment]
Hi @igorwulff we have updated our public develop branch and will create an RC release as soon as possible.

**Comment by tig-rikjonkmans** [comment]
v1.5.1-RC1 is released.

**Comment by richardreen** [comment]
Oke, installed that version and did all things like composer update and so...as described on your side...but all the time i get now this error:

{"0":"Please update your modules: Run \"composer install\" from the Magento root directory.\nThe following modules are outdated:\nTIG_PostNL db schema version: defined in codebase - 1.5.0, currently installed - 1.5.1-rc1\nTIG_PostNL db data version: defined in codebase - 1.5.0, currently installed - 1.5.1-rc1"

So is there something wrong in this RC1 version

**Comment by tig-rikjonkmans** [comment]
Hi @richardreen which description on our side do you refer to ?

Because the steps you should do are as followed:
1. open composer.json
2. change the require of tig/postnl-magento2 to 1.5.1-RC1
3. change stability to alpha
4. save composer.json and run composer update tig/postnl-magento2
5. run bin/magento setup:upgrade

When we run those on our test environments, it is going as expected and we can not reproduce the error you describe. 

Can you tell me what the values are of the TIG_PostNL record within your database table setup_module, this should be 1.5.0. Because we did not updated the setup version for this release. 
`<module name="TIG_PostNL" setup_version="1.5.0">`
https://github.com/tig-nl/postnl-magento2/blob/v1.5.1-RC1/etc/module.xml

**Comment by richardreen** [comment]
Currenly is in that table 1.5.1-rc1.
This version i got from Jeffrey because we had some problems 
Ticket:SD-14142) 05199317: Magento 2 -> upgrade naar versie 1.5 -> PostNL module loopt nu vast [ ref:_00Dw0CncL._5000J1LAPKN:ref ]
On  13-08-2018 jeffrey wrote
Jeffrey Branderhorst (TIG) commented:

Beste Richard,

Mijn collega heeft een fix hiervoor geschreven en op een public branch gezet. U kunt deze branch binnen halen en alvast testen als u wilt" https://github.com/tig-nl/postnl-magento2/tree/master-rc

Met vriendelijke groet / Kind regards / Mit freundlichen Grüßen / 最好的祝福,

Jeffrey Branderhorst
CUSTOMER CARE SPECIALIST

An that was version 1.5.1-rc1....so you publiced this version for the second time!!!!

**Comment by tig-rikjonkmans** [comment]
@richardreen the master-rc branch was never published as an RC official and was updated for the main purpose as described in your SD issue 14142. 

For now i have update the master-rc branch, so you could use that again if you like.

**Comment by richardreen** [comment]
Is it not better to get on the right track again...and leave the unofficial track? Otherwise i expect that these issues are coming back with future releases! I this possible on a simple way?

**Comment by tig-rikjonkmans** [comment]
@richardreen yes you are right, i was already typing a correction in my answer. The best thing you could do is change the db values back to 1.5.0 (schema and data).

**Comment by richardreen** [comment]
oke, lucky i did it on the staging server...so i can revert all back....i tried the first proposed with the update rc-branch,,, that worked but after that the upgrade to 2.2.6 is failing again because that updated branch is mising the 101.0.6
caused :+1:     - tig/postnl-magento2 dev-master-rc requires magento/framework >=100.1.0,<=100.1.14|>=101.0.0,<=101.0.5 -> satisfiable by magento/framework[101.0.5, 100.1.0-rc1, 100.1.0-rc2, 100.1.0-rc3, 100.1.0, 100.1.1, 100.1.2, 100.1.3, 100.1.4, 100.1.5, 100.1.6, 100.1.7, 100.1.8, 101.0.0-rc21, 101.0.0-rc22, 101.0.0-rc23, 101.0.0-rc30, 100.1.9, 101.0.0, 101.0.1, 100.1.10, 100.1.11, 101.0.2, 100.1.12, 101.0.3, 100.1.13, 101.0.4, 100.1.14].

Will restore the  backup and give it a new try with 1.5.0 in the table!!
